src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -12152,18 +12152,24 @@ class Test
             verifier.VerifyIL("Test.M02<T, U>()",
 @"
 {
-  // Code size       24 (0x18)
+  // Code size       39 (0x27)
```

Comment:
nit: I would probably have followed the pattern you used elsewhere: lock existing tests that affected to use `Regular10` (and duplicate some of them with `RegularNext`).

Reply:
This is another preview feature, not a C# 10 one.